.github/workflows/if-nodejs-pr-testing.yml

```
@@ -76,3 +76,6 @@ jobs:
       - if: steps.packagejson.outputs.exists == 'true'
         name: Run release assets generation to make sure PR does not break it
         run: npm run generate:assets --if-present
+      - if: steps.packagejson.outputs.exists == 'true'
+        name: Run JSON Schema Validator
+        run: npm run lint-schemas --if-present
```

no `--if-present` needed here
```suggestion
        run: npm run lint-schemas
```

---

resolved with 8399eb9